api/v1alpha1/dataset_types.go

```diff
@@ -169,6 +170,10 @@ type DatasetSpec struct {
 	// SharedEncryptOptions is the encryptOption to all mount
 	// +optional
 	SharedEncryptOptions []EncryptOption `json:"sharedEncryptOptions,omitempty"`
+
+	// StorageCapacity is the storage size of generated PVC and PV
+	// +optional
+	StorageCapacity resource.Quantity `json:"storageCapacity,omitempty"`
```

Comment:
Do we have to add a new field to the Dataset CRD? `StorageCapacity` is just informational in Fluid, might be confusing with some other concept like `tieredstore.quota`. How about following [PR#3146](https://github.com/fluid-cloudnative/fluid/pull/3146/files#diff-21714ad8e042b7d849b16c0100627db3ecd0f7230bcb59b2c58fcd535e0dd09aR177)? So we can override storage capacity with a given annotation on Dataset?

Reply:
Good point. Agree. I think we can use the annotation key `pvc.fluid.io/resources.request.storageSize` to specify the desired storage size for the PVC. This key is descriptive and aligns with the Kubernetes convention for resource requests. 

It can provide flexibility to extend the customization of PVC.

Reply:
ok,I will change it

Reply:
we have used “pvc.fluid.io/resources.requests.storage” as the annotation key on Dataset to  specify the desired storage size for the generated PVC.  If not set, or if the storage size format is incorrect, it will be set to the default value of 100Pi.